README.md

```diff
@@ -0,0 +1,117 @@
+# hadoop-yarn-api-python-client
+
+Python client for Apache Hadoop® YARN API
+
+[![Latest
+Version](https://img.shields.io/pypi/v/yarn-api-client.svg)](https://pypi.python.org/pypi/yarn-api-client/) [![Travis CI build
+status](https://travis-ci.org/toidi/hadoop-yarn-api-python-client.svg?branch=master)](https://travis-ci.org/toidi/hadoop-yarn-api-python-client) [![Latest documentation
+status](http://readthedocs.org/projects/python-client-for-hadoop-yarn-api/badge/?version=latest)](https://python-client-for-hadoop-yarn-api.readthedocs.org/en/latest/?badge=latest) [![Test
+coverage](https://coveralls.io/repos/toidi/hadoop-yarn-api-python-client/badge.png)](https://coveralls.io/r/toidi/hadoop-yarn-api-python-client)
+
+Package documentation:
+[python-client-for-hadoop-yarn-api.readthedocs.org](http://python-client-for-hadoop-yarn-api.readthedocs.org/en/latest/)
+
+REST API documentation: [hadoop.apache.org](http://hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-site/WebServicesIntro.html)
+
+Library is compatible with Hadoop 3.2.1.  
+If u have version other than mentioned, certain APIs might be not working or have differences in
+implementation. If u plan to use certain API long-term, you might want to make sure its not in Alpha stage in documentation.
+
+## Installation
+
+From PyPI
+```
+pip install yarn-api-client
+```
+
+From Anaconda (conda forge)
+```
+conda install -c conda-forge yarn-api-client
+```
+
+From source code
+```
+git clone https://github.com/toidi/hadoop-yarn-api-python-client.git
+pushd hadoop-yarn-api-python-client
+python setup.py install
+popd
+```
+
+##Enabling support for Kerberos/SPNEGO Security
+1. First option - using `requests_kerberos` package  
+
+To avoid deployment issues on a non Kerberized environment, the `requests_kerberos`
+dependency is optional and needs to be explicit installed in order to enable access
+to YARN console protected by Kerberos/SPNEGO.
+
+`pip install requests_kerberos`
+
+From python code
+```
+from yarn_api_client.history_server import HistoryServer
+from requests_kerberos import HTTPKerberosAuth
+history_server = HistoryServer('https://127.0.0.2:5678', auth=HTTPKerberosAuth())
+```
+
+PS: You __**need**__ to get valid kerberos ticket in systemwide kerberos cache before running your code, otherwise calls to kerberized environment won't go through (run kinit before proceeding to run code)
+
+2. Second option - using `gssapi` package  
+
+If you want to avoid using terminal calls, you have to perform SPNEGO handshake to retrieve ticket yourself. Full API documentation: https://pythongssapi.github.iQ/python-gssapi/latest/_
+
+#Usage
```

Comment:
Probably need a space after the hash mark.

Reply:
Resolved